### PR TITLE
MBS 7669 Update IDE speedup recommendations

### PR DESCRIPTION
**Pull request by eugene-krivobokov**
I've checked IDE recommendations on a new setup.

### CODE REVIEW

File: docs/content/docs/ide/Speedup.md [hunk 1]
```diff
@@ -3,134 +3,140 @@ title: IDE Speedup
...
+ 
+**Preferences > Editor > File Types > Ignore Files and Folders**
 
-## Освободи в ОС больше памяти
-
-Посмотри какие приложения потребляют много памяти, не используется ли swap.
+{{< hint warning>}}
+Some intermediate build files are still needed for development.
 
-Для Google Chrome есть плагины для авто-остановки старых вкладок: [The Great suspender](https://chrome.google.com/webstore/detail/the-great-suspender/klbibkeccnjlkjkiokjodocebajanakg?hl=en)
```
Comment (eugene-krivobokov) [inline docs/content/docs/ide/Speedup.md:127]: Chrome has a built-in tab discarding feature

File: docs/content/docs/ide/Speedup.md [hunk 2]
```diff
@@ -3,134 +3,140 @@ title: IDE Speedup
...
+{{< tab "Windows" >}}
 
-## kotlin.use.ultra.light.classes (experimental)
+[Add an exclusion to Windows Security](https://support.microsoft.com/en-us/help/4028485/windows-10-add-an-exclusion-to-windows-security)
 
-A light class is a representation of a Kotlin class as the Java PSI, allowing IntelliJ IDEA's Java support features to work with Kotlin classes.
+{{< /tab >}}
+{{< /tabs >}}
 
-`Cmd + Shift + A` - найди Registry, включи флаг `kotlin.use.ultra.light.classes`.
```
Comment (eugene-krivobokov) [inline docs/content/docs/ide/Speedup.md:120]: When I checked, It was enabled by default

**Comment by sergioboi** [comment]
Did you simply translate documentation to English?

**Review by sergioboi — APPROVED** [review]

**Comment by eugene-krivobokov** [comment]
> Did you simply translate documentation to English?

I translated it and removed obsolete parts.

**Pull request merged**